barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -140,6 +140,13 @@ bool proveAndVerify(const std::string& bytecodePath, const std::string& witnessP
  */
 bool accumulateAndVerifyGoblin(const std::string& bytecodePath, const std::string& witnessPath)
 {
+    // TODO(https://github.com/AztecProtocol/barretenberg/issues/811): Don't hardcode dyadic circuit size. Currently set
```

Comment:
GoblinAcirComposer initialises Goblin which calls the `perform_op_queue_interactions_for_mock_first_circuit` function so the crs needs to be initalised prior to this. Not sure if there's a better way to achieve this